### ISSUE TITLE
Self containned resources

### ISSUE THREAD

**Issue by Mujaja** [HelloZeroNet/Documentation#23]
Currently loading the site (from the ZeroNet clone or readthedocs) asks for resources from `https://fonts.googleapis.com`, `https://fonts.gstatic.com` and `http://zeronet.disqus.com`.

Wouldn't it be more zeronet-like to use internal fonts/resources? That wouldn't be possible with disqus, but can't we replace it with ZeroTalk and/or ZeroChat?

**Comment by HelloZeroNet** [comment]
The problem is the theme is not provided by me, so cant edit the resources location.

**Comment by Mujaja** [comment]
The theme can be uploaded as well. I just pushed it [here](https://github.com/Mujaja/Documentation/commit/2a84393b3a1f0652540d17bb142cd831432d1fb9). It weights 1.2 MB and the only other external resource is *disqus*, which I removed in [here](https://github.com/Mujaja/Documentation/commit/32a29424e056a5ce4b0873da0414f265a871cc04), though I'm not sure if simply deleting the docs.js file would be the correct way to go.

If you agree with any of those I can make a pull request in no time. Thoughts?
